amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -1701,33 +1701,34 @@ extern "C"
 	 * \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_UINT8</tt> or <tt>\ref VX_TYPE_FLOAT32</tt> or <tt>\ref VX_TYPE_FLOAT16</tt> or <tt>\ref VX_TYPE_INT8</tt> format data.
 	 * \param [in] pSrcRoi The input tensor of batch size in <tt>unsigned int</tt> containing the roi values for the input in xywh/ltrb format.
 	 * \param [out] pDst The output tensor in <tt>\ref VX_TYPE_UINT8</tt> or <tt>\ref VX_TYPE_FLOAT32</tt> or <tt>\ref VX_TYPE_FLOAT16</tt> or <tt>\ref VX_TYPE_INT8</tt> format data.
-	 * \param [in] pRainValue The input array in <tt>\ref VX_TYPE_FLOAT32</tt> format containing the rain value data.
+	 * \param [in] rainPercentage The input value in <tt>\ref VX_TYPE_FLOAT32</tt> format containing the rain value data.
```

Comment:
Change the descriptions accordingly for arguments. Some have been changed from array to scalar.